### PR TITLE
add some hints for troubleshooting with alpine container

### PR DESCRIPTION
**Pull request by kurtfm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,6 +152,13 @@ Basic information about the Agent execution are available through the `logs` com
 
 `docker logs dd-agent`
 
+Exec a shell on the container and tail logs for debugging (alpine version shown)
```
Comment (hkaj) [inline README.md]: could you add the debian version too while you're at it?

```
docker exec -it dd-agent bash
# tail -f /var/log/datadog/dogstatsd.log
```

File: README.md [hunk 2]
```diff
@@ -149,6 +152,13 @@ Basic information about the Agent execution are available through the `logs` com
...
+
+```
+$ docker exec -it dd-agent ash 
+/opt/datadog-agent # tail -f /opt/datadog-agent/logs/dogstatsd.log
```
Comment (hkaj) [inline README.md]: and also a note saying that logs from the other components of the agents are in the same folder collector.log, forwarder.log and jmxfetch.log. supervisor.log too but we already get this one from `docker logs dd-agent`.

File: README.md [hunk 3]
```diff
@@ -133,6 +133,9 @@ To display information about the Agent's state with this command.
 
 `docker exec dd-agent service datadog-agent info`
 
+On the alpine container...
```
Comment (hkaj) [inline README.md]: `On the alpine container:`

**Comment by hkaj** [comment]
Hi @kurtfm 
Thanks a lot for this update, I hope you didn't struggle too much to find these yourself.
I added a few comments. Once they are addressed we'll merge.
Thanks again!

**Comment by kurtfm** [comment]
Pushed some more changes per your feedback.

I didn't struggle too much... was having more difficulties with linking than anything else, but that was more my fault/env.

**Comment by hkaj** [comment]
Thanks for the changes, LGTM now.

About the linking, not sure if your environment allows it but the agent offers service discovery now. It could help you monitor apps/components running in other containers. Here are the [blogpost](https://www.datadoghq.com/blog/service-discovery-monitoring-docker-containers-move/) and [doc](http://docs.datadoghq.com/guides/servicediscovery/) about it in case you're interested.

**Pull request merged**